src/main/java/dev/aziz/movies/controllers/MovieController.java

```diff
@@ -27,4 +34,19 @@ public ResponseEntity<List<ReducedMovieDto>> getAllMovies() {
     public ResponseEntity<MovieDto> getMovie(@PathVariable Long id) {
         return new ResponseEntity<>(movieService.findMovieById(id), HttpStatus.OK);
     }
+
+    @PostMapping("/movies")
+    public ResponseEntity<MovieDto> createMovie(@RequestBody @Valid MovieDto movieDto) {
+        return new ResponseEntity<>(movieService.createMovie(movieDto), HttpStatus.CREATED);
+    }
+
+    @DeleteMapping("/movies/{id}")
+    public ResponseEntity<MovieDto> deleteMovie(@PathVariable Long id) {
+        return new ResponseEntity<>(movieService.deleteById(id), HttpStatus.OK);
```

Comment:
You can use the following syntax: `ResponseEntity.ok(movieService.deleteById(id)).build();` instead of indicating the Http code each time